server/containers/README.md

```diff
@@ -0,0 +1,123 @@
+|								|							|
+| ZOOKEEPER_NODE_LIST			| localhost:2181			| <i>comma separated list</i>
+| 								| 							|
+| SQL_PROVIDER_NAME				| mariadb 					| mariadb , postgresql
+| JDBC_HOST						| localhost					|
+| JDBC_PORT						| if mariadb: 3306<br>if postgresql: 5432|
+| JDBC_USERNAME					| sqladmin					|
+| JDBC_PASSWORD					| admin						|
+| JDBC_DB_NAME					| kaa 						|
+ 								| 							|
+| CASSANDRA_CLUSTER_NAME		| Kaa Cluster 				|
+| CASSANDRA_KEYSPACE_NAME		| kaa 						|
+| CASSANDRA_NODE_LIST			| localhost:9042 			| <i>comma separated list</i>
+| CASSANDRA_USE_SSL				| false 					|
+| CASSANDRA_USE_JMX				| true 						|
+| CASSANDRA_USE_CREDENTIALS		| false 					|
+| CASSANDRA_USERNAME 			| (empty) 					|
+| CASSANDRA_PASSWORD 			| (empty) 					| 
+| 								| 							| 
+| MONGODB_NODE_LIST 			| localhost:27017 			| 
+| MONGODB_DB_NAME				| kaa 						| 
+| MONGODB_WRITE_CONCERN 		| acknowledged 				| 
+| 								| 							| 
+| NOSQL_PROVIDER_NAME			| mongodb 					| mongodb , cassandra
+|								|							|
+| CONTROL_SERVER_ENABLED		| true						| true/false
+| BOOTSTRAP_SERVER_ENABLED		| true						| true/false
+| OPERATIONS_SERVER_ENABLED		| true						| true/false
+| THRIFT_HOST					| localhost					| 
+| THRIFT_PORT					| 9090						| 
+| ADMIN_PORT					| 8080						| 
+| SUPPORT_UNENCRYPTED_CONNECTION| true						| true/false
+| TRANSPORT_BIND_INTERFACE		| 0.0.0.0					| 
+| TRANSPORT_PUBLIC_INTERFACE	| current public host					|
+| METRICS_ENABLED				| true 						| true/false
+ 
+## Steps to deploy
+
+1. Download debian package from [official site](http://www.kaaproject.org/download-kaa/)
+ or build your Kaa project locally and copy kaa-node.deb from **server/node/target/** into **server/containers/docker/** folder.
+
+2. From server/containers/docker folder execute:
+```sh
+$ docker build --build-arg setupfile=kaa-node.deb -t kaa-node:0.10.0 .
+```
+or
+```sh
+$ sh build.sh
+```
+3. From server/containers/docker/using-compose folder run command:
+```sh
+$ python launch-kaa.py SQL-NoSQL
+```
+And specify SQL-NoSQL databases which you wan to use
+All available options of SQL-NoSQL databases:
+    * mariadb-mongodb
+    * mariadb-cassandra
+    * postgresql-mongodb
+    * postgresql-cassandra 
+If you want deploy Kaa in cluster mode, run previous command with additional optional
+```sh
+$ python launch-kaa.py SQL-NoSQL NODE_COUNT
+```
+4. Execute following command to get into running container:
+```sh
+$ docker exec -it usingcompose_KAA_SERVICE_NAME_1 /bin/bash -c "export TERM=xterm; exec bash"
+```
+
+Wait a few seconds until the service kaa-node starts.
+
+Open your browser on localhost:8080 (You can see there some error, just refresh page. It's mean that's service kaa-node does not start yet. 
+You can verify this by looking logs of kaa-node containers.)
+
+In this example usingcompose - it's name of folder where located docker-compose.yml files.
```

Comment:
it is not clear what docker-compose.yml we are talking about. Please clarify this in instruction. I would replace usingcompose_KAA_SERVICE_NAME_1 with something clear (cluster_compose_folder or similar)